web/chat/chat.react.js

```diff
@@ -3,13 +3,14 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 
-import ChatThreadList from './chat-thread-list.react';
 import ChatMessageList from './chat-message-list.react';
 import InputStateContainer from '../input/input-state-container.react';
+import ChatTabs from './chat-tabs.react';
 
 type Props = {|
   setModal: (modal: ?React.Node) => void,
 |};
+
```

Comment:
Nit: I've noticed you like to add newlines here. I'd prefer to leave this class as-is